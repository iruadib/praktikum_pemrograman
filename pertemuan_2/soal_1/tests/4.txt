Jane Doe
100000000 1 1