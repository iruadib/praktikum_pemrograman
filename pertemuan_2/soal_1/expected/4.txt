Payslip for Employee
---------------------
Name: Jane Doe
Tax (20%): Rp20.000.000,00
Installment: Rp1,00
Insurance: Rp1,00
Net Salary: Rp79.999.998,00
